packages/esm-appointments-app/src/appointments-header/appointments-header.component.tsx

```diff
@@ -51,24 +51,24 @@ const AppointmentsHeader: React.FC<AppointmentHeaderProps> = ({ title, onChange
             />
           </DatePicker>
         </div>
-        {typeof onChange === 'function' && (
-          <div className={styles.dropdownContainer}>
+        <div className={styles.dropdownContainer}>
```

Comment:
Well done @subhamkumarr  ,   @denniskigen . Do we  still need the View Button basing on this fix. Looks like it does nothing after this fix.

Reply:
@sherrif10 sir i didn't get you. My PR is not good to merge?